ga4gh/datamodel/genotype_phenotype.py

```diff
@@ -39,22 +39,18 @@ def __init__(self, parentContainer, localId, randomSeed):
         super(SimulatedPhenotypeAssociationSet, self).__init__(
             parentContainer, localId)
 
-    # TODO this doesn't make much sense
     def getAssociations(
             self, request=None, featureSets=[]):
-        if request:
-            fpa = protocol.FeaturePhenotypeAssociation()
-            fpa.phenotype_association_set_id = self._parentContainer.getId()
-            fpa.id = ""
-            fpa.feature_ids.extend(['featureId'])
-            fpa.evidence.extend([protocol.Evidence()])
-            fpa.phenotype.MergeFrom(protocol.PhenotypeInstance())
-            fpa.description = "description"
-            fpa.environmental_contexts.extend(
-                [protocol.EnvironmentalContext()])
-            return [fpa]
-        else:
-            return []
+        fpa = protocol.FeaturePhenotypeAssociation()
+        fpa.phenotype_association_set_id = self._parentContainer.getId()
+        fpa.id = ""
+        fpa.feature_ids.extend(['featureId'])
+        fpa.evidence.extend([protocol.Evidence()])
+        fpa.phenotype.MergeFrom(protocol.PhenotypeInstance(id='phenotypeId'))
+        fpa.description = "description"
+        fpa.environmental_contexts.extend(
+            [protocol.EnvironmentalContext(id='environmentalContextId')])
+        return [fpa]
```

Comment:
So the simulated `getAssociations` returns an array of length one in all cases?  That's not how the non-simulated `getAssociations` works, does it?  We want the operation of both to be as close as possible -- the point here is to simulate the return of real data, without having to load real data in for testing.  See `ga4gh/datamodel/reads.py` for a good example.

We can also vary what gets returned by use of a random number generator (see the previously mentioned example), but if and how we use that depends on what data we want to simulate and what conditions we want to test.  You know the data model better than me, so that's your call.

Another thing we want to do is to allow the user to control the simulated variables in the server config.  I just pushed a PR that does this for RNA: #1349  You should create variables in `serverconfig.py` that determine e.g. the number of `SimulatedPhenotypeAssociationSet`s that get created (I always like 2 as a default) in the simulated configuration.